.evergreen/scripts/send-perf-data.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+# We use the requester expansion to determine whether the data is from a mainline evergreen run or not
+if [ "${requester}" == "commit" ]; then
```

Comment:
```suggestion
if [ "${requester:?}" == "commit" ]; then
```
Recommend using `${var:?}` form for all variable expansions to error on unset or null. For variables which may be set but null, recommend explicitly documenting via the `"${var:-}"` form instead. ([SC2154](https://www.shellcheck.net/wiki/SC2154))